cmd/elastic.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e;
+
+function elastic_schema_drop(){ compose_run 'schema' node scripts/drop_index "$@" || true; }
+function elastic_schema_create(){ compose_run 'schema' npm run create_index; }
+function elastic_start(){ compose_exec up -d elasticsearch; }
+function elastic_stop(){ compose_exec kill elasticsearch; }
+
+register 'elastic' 'drop' 'delete elasticsearch index & all data' elastic_schema_drop
+register 'elastic' 'create' 'create elasticsearch index with pelias mapping' elastic_schema_create
+register 'elastic' 'start' 'start elasticsearch server' elastic_start
+register 'elastic' 'stop' 'stop elasticsearch server' elastic_stop
+
+# to use this function:
+# if test $(elastic_status) -ne 200; then
+function elastic_status(){ curl --output /dev/null --silent --write-out "%{http_code}" "http://${ELASTIC_HOST:-localhost:9200}" || true; }
```

Comment:
Here is where the localhost comes from

Reply:
Oops, never mind, this is run _outside_ a docker container, so localhost is correct